legacy_packages/sdk/src/evm/core/classes/internal/marketplace/marketplace-auction.ts

```diff
@@ -254,6 +254,9 @@ export class MarketplaceAuction {
             startTime: BigNumber.from(listingStartTime),
           } as IMarketplace.ListingParametersStruct,
         ],
+        overrides: {
+          extraGas: 50_000n,
```

Comment:
I believe the `_` notation breaks some older bundlers, to be super safe can we do `BigInt(50000)` everywhere here